packages/server/graphql/mutations/helpers/isPatientZero.ts

```diff
@@ -1,6 +1,16 @@
 import getPatientZeroByDomain from '../../../postgres/queries/getPatientZeroByDomain'
 
-const isPatientZero = async (userId: string, domain?: string | null) =>
-  domain ? userId === (await getPatientZeroByDomain(domain)).id : false
+const isPatientZero = async (userId: string, domain?: string | null) => {
+  if (domain) {
+    const patient0 = await getPatientZeroByDomain(domain)
+    if (!patient0) {
```

Comment:
Fix a bug where when the new user is actually indeed a patient 0, `getPatientZeroByDomain(domain)` would return `undefined` and that causes accessing `.id` throw an error.